Samples/iOS-Swift/PerformanceBenchmarks/SentrySDKPerformanceBenchmarkTests.m

```diff
@@ -10,7 +10,7 @@ @implementation SentrySDKPerformanceBenchmarkTests
 
 - (void)testCPUBenchmark
 {
-    XCTSkipIf(isSimulator() && !isDebugging());
+    //    XCTSkipIf(isSimulator() && !isDebugging());
```

Comment:
m: IMO commented code without an explanation has no value.
Maybe this was forgotten here by mistake.
We should either remove this or add an explanation.

Reply:
Oops, yes this was a mistake